src/ui/components/Library/Navigation/Navigation.tsx

```diff
@@ -17,8 +17,8 @@ function Teams() {
 
   return (
     <>
-      {workspaces.map((w, i) => (
-        <TeamButton label={w.name} key={i} id={w.id} isTest={w.isTest} />
+      {workspaces.map(w => (
+        <TeamButton label={w.name} key={w.id} id={w.id} isTest={w.isTest} />
```

Comment:
This is probably not a big deal, but we should not use the index of the list item as the key. It's unlikely this list will change order, but if it does, using indices as keys will create bugs.